### ISSUE TITLE
Setting 'force_timeout = -1' never kills sssd_be/sssd_nss process

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#1626]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/1622
- Created at 2012-11-08 11:24:33 by [jhrozek](https://pagure.io/user/jhrozek)
- Closed as Duplicate
- Assigned to nobody
---

https://bugzilla.redhat.com/show_bug.cgi?id=874474 (*Red Hat Enterprise Linux 6*)

    Description of problem:
    When force_timeout parameter is set to -1 in the DOMAIN/NSS section of
    sssd.conf, the backend process never gets killed. Upon verifying the sssd log
    files, we can see "Killing service [LDAP]" notification however "Sending
    SIGKILL" never notification never appears. Either we should run SIGCONT to get
    sssd_be/sssd_nss process restarted or kill all processes explicitly and start
    from scratch.
    
    Version-Release number of selected component (if applicable):
    sssd-1.9.2-7.el6.x86_64
    
    How reproducible:
    Always
    
    Steps to Reproduce:
    1. Configure sssd.conf as given below:
    
    [sssd]
    reconnection_retries = 3
    config_file_version = 2
    sbus_timeout = 30
    services = nss, pam
    domains = LDAP
    debug_level = 0x0270
    
    [nss]
    filter_groups = root
    filter_users = root
    reconnection_retries = 3
    
    [pam]
    reconnection_retries = 3
    
    [domain/LDAP]
    id_provider = ldap
    auth_provider = ldap
    enumerate = FALSE
    cache_credentials = TRUE
    debug_level = 9
    ldap_uri = ldap://SERVER
    ldap_search_base = dc=example,dc=com
    ldap_tls_cacert = /etc/openldap/certs/cacert.asc
    force_timeout = -1
    
    2. Start sssd service and send SIGSTOP to process sssd_be
       # kill -s SIGSTOP <PID>
    
    3. Monitor sssd.log file for the messages.
       # tail -f /var/log/sssd.log
    
    
    Actual results:
    The sssd service restarts successfully and "Killing service [LDAP]"
    notification was seen however, sssd_be never gets killed. We should run SIGCONT
    to get sssd_be process restarted.
    
    Expected results:
    Either a warning message while restarting the service OR service should never
    start.
    
    Additional info:

### Comments

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2012-11-08 15:36:27](https://pagure.io/SSSD/sssd/issue/1622#222984)**

Fields changed

blockedby:  => 
blocking:  => 
coverity:  => 
design:  => 
design_review:  => 0
feature_milestone:  => 
fedora_test_page:  => 
milestone: NEEDS_TRIAGE => SSSD Deferred
testsupdated:  => 0

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2012-11-09 01:56:03](https://pagure.io/SSSD/sssd/issue/1622#222985)**

Fields changed

blocking:  => [https://fedorahosted.org/sssd/ticket/416 416]

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2012-11-09 01:57:26](https://pagure.io/SSSD/sssd/issue/1622#222986)**

Fields changed

blocking: [https://fedorahosted.org/sssd/ticket/416 416] => [https://fedorahosted.org/sssd/ticket/416 416]

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2015-04-24 16:03:44](https://pagure.io/SSSD/sssd/issue/1622#222987)**

Closing as duplicate of #2269

changelog:  => 
mark:  => 0
resolution:  => duplicate
review:  => 0
selected:  => 
status: new => closed

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-24 14:27:10](https://pagure.io/SSSD/sssd/issue/1622#222988)**

**Metadata Update from @jhrozek**:
- Issue marked as blocked by: #416
- Issue set to the milestone: SSSD Patches welcome
